compose-builder/README.md

```diff
@@ -206,7 +209,7 @@ Options:
     asc-mqtt-s:  Pull includes App Service MQTT Export Secrets
     mqtt-broker: Pull includes MQTT Broker service
     mqtt-bus:    Pull includes additional service for MQTT Message Bus
-    ui:          Pulls only the EdgeX UI service image. `ds-x`, 'mqtt', 'no-ds' & 'no-secty' are ignored
+    ui:          Pulls only the EdgeX UI service image.
```

Comment:
do we need a `ui-only` here? what happens if i do a `make pull -ui-only`?

Reply:
Yea, apparently it is still available... ;-) Will add it back to the README. good catch!